tests/e2e/operation/multi.go

```diff
@@ -0,0 +1,223 @@
+// +build e2e
+
+//
+// Copyright (C) 2019-2021 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package operation
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+)
+
+func (c *client) MultiSearch(t *testing.T, ctx context.Context, ds Dataset) error {
+	client, err := c.getClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	cfg := &payload.Search_Config{
+		Num:     3,
+		Radius:  -1.0,
+		Epsilon: 0.1,
+	}
+
+	reqs := make([]*payload.Search_Request, 0, len(ds.Test))
+	for _, v := range ds.Test {
+		reqs = append(reqs, &payload.Search_Request{
+			Vector: v,
+			Config: cfg,
+		})
+	}
+
+	req := &payload.Search_MultiRequest{
+		Requests: reqs,
+	}
+	res, err := client.MultiSearch(ctx, req)
+	if err != nil {
+		return err
+	}
+	
+	if len(res.GetResponses()) != len(ds.Test) {
+		t.Error("number of responses does not match with sent requests")
+	}
+
+	return nil
+}
+
+func (c *client) MultiSearchByID(t *testing.T, ctx context.Context, ds Dataset) error {
+	client, err := c.getClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	cfg := &payload.Search_Config{
+		Num:     3,
+		Radius:  -1.0,
+		Epsilon: 0.1,
+	}
+
+	reqs := make([]*payload.Search_IDRequest, 0, len(ds.Test))
+	for i := range ds.Test {
+		reqs = append(reqs, &payload.Search_IDRequest{
+			Id:     strconv.Itoa(i),
+			Config: cfg,
+		})
+	}
+
+	req := &payload.Search_MultiIDRequest{
+		Requests: reqs,
+	}
+	_, err = client.MultiSearchByID(ctx, req)
```

Comment:
should we check the length also? (all of the functions in this file)

Reply:
other than this comment, LGTM for me

Reply:
Thanks! I added them.